pkg/compare/compare.go

```diff
@@ -0,0 +1,283 @@
+package compare
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+)
+
+type CompareFunc func(x, y interface{}) bool
+
+func defaultCompareMap() map[string]CompareFunc {
```

Comment:
@jirevwe Can this be a const? Can't think of a scenario where this will need to change.

Reply:
Oh yeah

Reply:
Hmm, apparently maps can't be made constants, I tried it and it didn't work.

Reply:
Maybe variable, then?